crates/db_views/src/comment_view.rs

```diff
@@ -927,6 +941,52 @@ mod tests {
     cleanup(data, pool).await;
   }
 
+  #[tokio::test]
+  #[serial]
+  async fn test_saved_order() {
```

Comment:
Added a test here, and I verified that it breaks when not doing the filter above.